src/supermarket/engines/fieri/app/models/cookbook_artifact.rb

```diff
@@ -107,7 +107,8 @@ def binary?(filepath)
     # need to check for application/xml explicitly because only versions of
     # libmagic < 5.26 are available on Ubuntu 14.04 and 16.04 which are our
     # CI (Travis and Automate) node platforms
-    Utils::FileFormat.get_mime_type(file_path: filepath) !~ %r{^(text\/|inode\/x-empty|application\/xml)}
+    # For ubuntu 22.04 we need to include 'application\/json' to match the json file
+    Utils::FileFormat.get_mime_type(file_path: filepath) !~ %r{^(text\/|inode\/x-empty|application\/xml|application\/json)}
```

Comment:
In **Ubuntu 22.04** MIME TYPE of any json file is being picked up perfectly (as `application/json`) but not in **Ubuntu 20.04** (as `text/plain`) so we need to include `application\/json` here in order to decide whether the file passed in this method is binary or not.